tests/integration/analytics/test_gather_analytics.py

```diff
@@ -124,6 +126,7 @@ def test_gather_analytics_invalid_settings(
     ],
 )
 @pytest.mark.django_db
+@override_settings(FLAGS={"EDA_ANALYTICS": [("boolean", True)]})
```

Comment:
Is there a better way to override only flag `EDA_ANALYTICS`, consider FLAGS in the future can expand?

Reply:
There are a few ways to do this via unit testing effectively that I've found - 
1. override-settings: https://github.com/cfpb/django-flags/blob/main/flags/tests/test_urls.py#L132
   a. Can also do this via this [syntax](https://github.com/ansible/eda-server/pull/1132/files#diff-e3e0c48a81c019581f7b22fe7d99bb2c712d7460877558335201d913daa193d6R37-R39)
2. FlagCreate: https://github.com/cfpb/django-flags/blob/main/flags/tests/test_models.py#L8-L10
3. Via django-admin cli command - https://github.com/ansible/eda-server/pull/1132/files#diff-e3e0c48a81c019581f7b22fe7d99bb2c712d7460877558335201d913daa193d6R37-R39
   a. https://cfpb.github.io/django-flags/management_commands/
   
 Not entirely sure which is the best currently, but open to perspectives here

Reply:
After some further thought, I can probably define a generic function to set feature flags, to be used in unit test, which can take in an array of feature flags to set their state automatically via environment variable directly as well

Reply:
Tried to implement this, but ran into some hurdles here. Wondering if we have a way to refresh the settings after updating environmental variables?
For unit testing, I expect way may be able to override the flags directly via override settings for the tests that would require this, but open to other perspectives here

Reply:
@zkayyali812 
We can have something like [this](https://github.com/ansible/ansible-rulebook/blob/7c1840b17b6934c804b5cf47fc62489eb1cb2780/tests/e2e/conftest.py#L8), but modiying direclty `os.environ` Anyway this might be tricky and we need to take care of ensuring it doesn't not affect to preloaded entities like dynaconf settings, so I guess it won't work in some cases. 
I think it is fine as it is, we can rely on dynaconf for envvars handling.